src/pages/steps/AdditionalInformationStep/OrganizerPicker.tsx

```diff
@@ -272,6 +281,7 @@ const OrganizerPicker = ({
                 {addButtonHasBeenPressed && (
                   <Typeahead<Organizer>
                     options={organizers}
+                    flex-grow="1"
```

Comment:
This won't have any effect since the prop doesn't exist on `Typeahead` right?

if you want to use flex-grow on here. You can add it to the `Box` component like `flexShrink`, but it should be called `flexGrow`